settings.gradle.kts

```diff
@@ -7,6 +7,20 @@ File(rootDir, "lib").eachDir {
     project(":lib-$libName").projectDir = File("lib/$libName")
 }
 
+// Fix deprecation warnings with Gradle 8.5+.
+// See https://docs.gradle.org/8.5/userguide/upgrading_version_8.html#deprecated_missing_project_directory
+listOf(
+    ":extensions" to "$rootDir/gradle", // Temporary workaround.
+    ":extensions:individual" to "$rootDir/src",
+    ":extensions:multisrc" to "$rootDir/generated-src",
+).forEach { (name, path) ->
+    val projectDir = file(path)
+    if (projectDir.exists()) {
+        include(name)
+        project(name).projectDir = projectDir
+    }
+}
+
```

Comment:
I think we'll rename the projects after we migrate to the new multisrc structure so this is probably not needed.

NB: we can probably just conform to the Gradle default and use directory paths as project names, i.e. `:lib:dataimage` instead of `:lib-dataimage`.

Reply:
Just revert this part.